packages/components/src/components/static-url/static-url.jsx

```diff
@@ -1,8 +1,10 @@
 import React from 'react';
-import { getStaticUrl, PlatformContext } from '@deriv/shared';
+import { getStaticUrl, PlatformContext, setUrlLanguage } from '@deriv/shared';
+import { getLanguage } from '@deriv/translations';
 
 const StaticUrl = ({ href, is_document, children, ...props }) => {
     const { is_appstore } = React.useContext(PlatformContext);
+    setUrlLanguage(getLanguage());
 
     return (
         <a href={getStaticUrl(href, { is_appstore }, is_document)} rel='noopener noreferrer' target='_blank' {...props}>
```

Comment:
<a href={getHref()} rel='noopener noreferrer' target='_blank' {...props}>

Reply:
The anchor tag is what the StaticUrl component renders. So does it make a diff between your suggestion and @yauheni-kryzhyk-deriv's implementation?

Reply:
No, there is no differences, just a little bit different approach.